### PR TITLE
Parameters Pass for "Optimal Savings III: Occasionally Binding Constraints"

### PR DESCRIPTION
**Pull request by arnavs**
Not much going on in this one; there was one struct that wasn't mutated. 

#135

### CODE REVIEW

**Comment by Nosferican** [comment]
See if everything is good. Made a note of the missing maximize pass.

File: rst_files/ifp.rst [hunk 1]
```diff
@@ -48,18 +48,11 @@ This method turns out to be
 
 * More efficient numerically than value function iteration
 
-
 References
 ----------------
 
 Other useful references include :cite:`Deaton1991`, :cite:`DenHaan2010`, :cite:`Kuhn2013`, :cite:`Rabault2002`,  :cite:`Reiter2009`  and :cite:`SchechtmanEscudero1977`
```
Comment (arnavs) [inline rst_files/ifp.rst]: @Nosferican I think the convention is to put the setup right at the end of the overview section, as opposed to before the first code block. But I don't have strong feelings about it.

Comment (Nosferican) [reply rst_files/ifp.rst]: It does make working the lectures a pain. In the Setup discussion I had indicated to have them right before the `using Test` (right before any code execution). The Setup is only relevant to executing code, why should it be positioned between a bunch of markdown cells and actual code? It also forces to go up to the top to run that one cell before being able to work with the code.

Comment (arnavs) [reply rst_files/ifp.rst]: Yeah, I'm fine with either. Let's see what @jlperla thinks.

**Comment by Nosferican** [comment]
I am arguing that the Setup include should be right before the `using Test` cell (right before the first code cell) instead of currently where it stands before the markdown cells explaining the lesson and the start of the code cells.

**Comment by arnavs** [comment]
@Nosferican checked with Jesse, and I don't think he minds too much. So this should be fine. Will merge now.

**Pull request merged**